addons/es/verifactu/extensions.go

```diff
@@ -275,9 +274,13 @@ var extensions = []*cbc.Definition{
 				alongside the ~es-verifactu-op-class~ extension. Values correspond to the
 				L10 list.
 
-				No attempt will be made by GOBL to automatically map tax rates to exemption
-				reason codes, they will need to be determined and applied on a case-by-case
-				basis.
+				There are 2 scenarios where exempt extensions can be automatically assigned:
```

Comment:
We shouldn't determine tax combo values based on context, the definitions should be able to handle the different scenarios directly. This is important as Tax Combos need to be atomic.